app/apps/openzaak/helpers.py

```diff
@@ -41,6 +41,26 @@ def _parse_date(date):
     return None
 
 
+def _get_description(id, zaaktype_identificatie):
+    """Get the description based on the zaaktype_identificatie."""
+    if zaaktype_identificatie == settings.OPENZAAK_ZAAKTYPE_IDENTIFICATIE_HANDHAVEN:
+        return f"""
+        Het dossier ({id}) is nu in de handhavingsfase. Het staat nog niet vast dat gehandhaafd zal worden.
+        Uit de onderzoeksresultaten kan bijvoorbeeld volgen dat de woning regulier bewoond wordt
+        en dus niet gehandhaafd moet worden, dat het dossier terug moet naar de toezichtfase omdat
+        de woning opnieuw bezocht moet worden door de toezichthouders of dat een andere actie benodigd is.
+        Handhaving begint eerst met een aanschrijving waarin wij kenbaar maken voornemens zijn om handhavend
+        op te treden. Daarop kan een zienswijze worden ingediend. Vervolgens zullen wij nogmaals naar alle
+        feiten en omstandigheden kijken en overwegen of een handhavingsbesluit moet worden opgelegd,
+        of dat daarvan moet worden afgezien.
+        """
+    return f"""
```

Comment:
Ik hoef deze code niet te onderhouden, maar deze twee `return f"""` vlak na elkaar moest ik twee keer lezen. Aanname: het is een `if OPENZAAK_ZAAKTYPE_IDENTIFICATIE_HANDHAVEN return '...in handhavingsfase...' [else if] return '...toezichtsfase...'` klopt dat?

Zou er even een witregeltje tussen plakken :)

Reply:
Klopt! Door de `return` is de `else` overbodig. https://pylint.readthedocs.io/en/stable/user_guide/messages/refactor/no-else-return.html